pkg/controller/actions/render/render_metrics.go

```diff
@@ -0,0 +1,32 @@
+package render
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	// RenderedResourcesTotal is a prometheus counter metrics which holds the total
+	// number of resource rendered by the action per controller and rendering type.
+	// It has two labels.
+	// controller label refers to the controller name.
+	// engine label refers to the rendering engine.
```

Comment:
guess for now we only use kustomize as engine
and the controller is the CR's kind not name e.g Dashboard not default-dashboard.

Reply:
yep correct